composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.7",
-        "arbor-education/php-coding-standard": "@dev",
+        "arbor-education/php-coding-standard": "^1.0",
```

Comment:
@IsaacStarkie  I think we are missing `composer.lock` file here also

Reply:
Please see my reply to Davor. I can add a brand new lock file but would have to remove it to .gitignore